### ISSUE TITLE
Gold Income

### ISSUE THREAD

**Issue by Simpada** [shirograhm/TooManyItemsRoR2#1]
When I have this mod enabled, the amount of gold i earn is vastly increased, I easily get several thousand gold on the first stage after only a few minutes.

**Comment by Cheeseless** [comment]
I can corroborate this issue:

When I kill enemies, I get way more gold than expected. Specifically tested as such: only installing TooManyItems in r2modman, loading up the game with default difficulty, then killing only the first Wisp or small Lemurian within the first 30 seconds. In unmodded vanilla my total gold goes to 16. With TooManyItems my total gold goes to 41. 

I also tried running just the dependencies of TooManyItems to see if they were causing it, but they don't change the gold gain.

Based on the mod description there's no indication that this should happen. I tried to look through the code on Github but I'm not sufficiently familiar with ROR2 modding to see anything suspicious.
